server/index.js

```diff
@@ -1,4 +1,5 @@
 const app = require('./app');
+
 const port = 8080;
 
-app.listen(port, () => console.log(`Example app listening at http://localhost:${port}`));
```

Comment:
you shouldn't have removed the arrow function.
now it prints the log before the actual listening starts.

Reply:
Fixed thank you.